CMakeLists.txt

```diff
@@ -150,7 +150,7 @@ add_subdirectory(third_party/pybind11)
 add_subdirectory(third_party/rapidcheck)
 
 if (ONNX_MLIR_ENABLE_MHLO)
-  add_subdirectory(third_party/mlir-hlo EXCLUDE_FROM_ALL)
```

Comment:
Why is it necessary to remove this? My expectation is that this should be fine since we don't want to build it as part of the `all` target and only when needed by other targets.